src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/bulkscan/AttachScannedDocsCallbackHandlerTest.java

```diff
@@ -0,0 +1,181 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.bulkscan;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
+import uk.gov.hmcts.cmc.ccd.domain.CCDClaimDocument;
+import uk.gov.hmcts.cmc.ccd.domain.CCDClaimDocumentType;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCollectionElement;
+import uk.gov.hmcts.cmc.ccd.domain.CCDScannedDocument;
+import uk.gov.hmcts.cmc.ccd.domain.CaseEvent;
+import uk.gov.hmcts.cmc.ccd.domain.defendant.CCDRespondent;
+import uk.gov.hmcts.cmc.ccd.domain.defendant.CCDResponseMethod;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.CallbackParams;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.CallbackType;
+import uk.gov.hmcts.cmc.claimstore.utils.CaseDetailsConverter;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.cmc.ccd.domain.defendant.CCDResponseMethod.OFFLINE;
+
+@ExtendWith(MockitoExtension.class)
+class AttachScannedDocsCallbackHandlerTest {
+
+    @Mock
+    private CaseDetailsConverter caseDetailsConverter;
+
+    private AttachScannedDocsCallbackHandler callbackHandler;
+
+    private CallbackParams callbackParams;
+
+    @BeforeEach
+    void setUp() {
+        callbackHandler = new AttachScannedDocsCallbackHandler(caseDetailsConverter);
+
+        CallbackRequest callbackRequest = CallbackRequest
+            .builder()
+            .caseDetails(CaseDetails.builder().data(Collections.EMPTY_MAP).build())
```

Comment:
`Collections.emptyMap()` is almost always preferred, as it has generic type inference.
And elsewhere in this PR